labs/src/main/scala/org/scalalabs/basic/lab02/CollectionExercise.scala

```diff
@@ -33,7 +33,11 @@ object CollectionExercise01 {
    *
    */
   def googleCodeJamGooglerese(lines: String*): Seq[String] = {
-    error("fix me")
+    val input = "aozejp mysljylc kd kxveddknmc re jsicpdrysirbcpc ypc rtcsra dkh wyfrepkym veddknkmkrkcdde kr kd eoya kw aej tysr re ujdr lkgc jv"
+    val output = "yeqour language is impossible to understandthere are twenty six factorial possibilitiesso it is okay if you want to just give up"
```

Comment:
At first this looked like an awkward way of doing this (as opposed to having each sample input and output as a separate string), but it looks like this might actually run just a tiny bit faster since you create one big map instead of three smaller ones that you combine.